source/_data/plugins.yml

```diff
@@ -2277,3 +2277,14 @@
     - options
     - parameters
     - customization
+- name: hexo-filter-image
+  description: 自动为hexo中的图片映射绝对路径 | Automatically maps absolute paths to images in hexo
+  link: https://github.com/JoeyBling/hexo-filter-image
+  tags:
+    - filter
+    - image
+    - path
+    - asset
+    - options
```

Comment:
please remove this tag and the rest, the plugin only have loggin option.

Reply:
I meant remove `options`, `parameters` and `customization` tags.

Reply:
> I meant remove `options`, `parameters` and `customization` tags.

I see you deleted it

Reply:
Well, I think it should be all right now.
Sorry, I rarely use it PR

Reply:
Please remove `options` tag as well.

Reply:
> Please remove `options` tag as well.

no problem,i already remove that's tag